### PR TITLE
Updates to all longform content pages (but mostly the blog pages)

### PR DESCRIPTION
**Pull request by chrisannetts**
An update to make the blog posts a little more desirable. The current blog post styles were a 'first stab' at moving away from the previous aesthetic, and this is a more considered approach.

Along the way I've:

**Main bits**
- A bigger font size, and a shorter line length.
- Better sized headings.
- Created a more structured layout, rather than the stacking of elements we had before.
- Fixed the share buttons, which were broken for Twitter and only half worked for Facebook.

**Bonus bits**
- Updated the headings of all 'content' pages so that the logo aligns with the logo on non-content pages.
- Updated the site theme colour (for Android users).
- Removed the gnarly white background on the navigation at smaller sizes.
- Fixed the 320px-wide overflow issue.

### CODE REVIEW

**Comment by CurtisALee** [comment]
From a design point of view, I really like these :)

Particularly like the sticky share icon (reminds me of Medium), and the link hover effects.

The changes are only subtle but they've made a big difference to them. Love them!

---

Also, what's the _helper file for? Is that for general layout helpers like `margin: 0 auto`, or along those lines?

File: src/base.pug [hunk 1]
```diff
@@ -67,6 +67,8 @@ html.no-js(lang="en-GB")
     meta(name="twitter:site" content="@we_are_leaf")
     meta(name="twitter:creator" content="@we_are_leaf")
     link(rel="stylesheet", href="/assets/styles/main.css")
+    link(href='https://fonts.googleapis.com/css2?family=Playfair+Display:ital@1&display=swap' rel='stylesheet')
```
Comment (michaelcarter) [inline src/base.pug:70]: Are we using this on every page? If not, should we configure it to only be included on the ones its needed on?

File: src/pages/blog/5-tips-for-actually-shipping-a-side-project.pug [hunk 2]
```diff
@@ -7,81 +7,86 @@ block prepend vars
...
-  +banner({
-  })
+  article
 
-  .banner-spacer
+    +hero({
+      heroHeading: heroHeading,
+      heroAuthor: "Mike Carter",
+      heroDate: "20 March 2017",
+      heroMinutesToRead: 4
```
Comment (michaelcarter) [inline src/pages/blog/5-tips-for-actually-shipping-a-side-project.pug:19]: Not a blocker, but isn't the "hero" prefix on the attributes being passed in here redundant? It's already implicit they're for the hero by the fact they're passed to the hero.

**Comment by chrisannetts** [comment]
> From a design point of view, I really like these :)
> 
> Particularly like the sticky share icon (reminds me of Medium), and the link hover effects.
> 
> The changes are only subtle but they've made a big difference. Love them!
> 
> Also, what's the _helper file for? Is that for general layout helpers like `margin: 0 auto`, or along those lines?

Thanks Curt! 😊

It's for exactly that sort of thing. Quite often the file would be named utilities. Common examples of reusable classes that might go in here are `.sr-only`, `.visually-hidden`, and `.mb-0`.

File: src/pages/blog/5-tips-for-actually-shipping-a-side-project.pug (hunk 2 above)
Comment (chrisannetts) [reply src/pages/blog/5-tips-for-actually-shipping-a-side-project.pug:19]: I've thought about this a few times, and it's for no better reason than it just follows the convention of the other components. If you're happy for us to, we'll do a sweep of all components and make their attribute names a little nicer.

File: src/base.pug (hunk 1 above)
Comment (chrisannetts) [reply src/base.pug:70]: Good spot. Currently just the blog pages, so I'll cap it to there.

**Pull request merged**